### PR TITLE
Create SMS Logs document

### PR DESCRIPTION
**Pull request by jpwalden**
Gathering of SMS logs doc created

### CODE REVIEW

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		* Type :wq to save and exit; tap 'Enter'
+
+	* To verify the change do:
+```
+cat /etc/systemd/journald.conf
+```
+
+* Reboot phone to put the changes into effect.
+
+* Install the oFono Logger application from Jolla Store.
```
Comment (jovirkku) [inline Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md:40]: Add link /Support/Help_Articles/Collecting_Logs/Collect_oFono_Logs/ here. Installing oFono Logger is different on different devices.

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	* Visit oFono Logger app again and save the log as instructed in this article.
+
+* Export journal output into a file:
+	
+	```
+	devel-su
+	journalctl -b -a --no-tail --no-pager > journal.txt
+	```
+
+* Please send the log files logcat.txt, journal.txt and ofono_*.tar.gz  from $HOME/Documents to Jolla for analysis. Mention the time of device when the problem happened.
```
Comment (jovirkku) [inline Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md:71]: Please find the log files logcat. txt and journal.txt in $HOME and ofono_*.tar.gz in $HOME/Documents. If you send them for analysis mention the exact time when the problem happened.

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+* Stop logging.
+
+	* Use ```ctrl-C``` at the terminal to stop collecting the radio log and to close file logcat.txt.
+	* Visit oFono Logger app again and save the log as instructed in this article.
+
+* Export journal output into a file:
+	
+	```
+	devel-su
```
Comment (jovirkku) [inline Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md:67]: This is on line 53 already. It seems that there was no reason to exit the root mode in the meanwhile, so we can probably leave this deve-su out.

**Review by jovirkku — CHANGES_REQUESTED** [review]
Looks good to me. 
But I proposed some small changes.

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md]: Good point, link is there below, but makes sense to add it there as well > added.

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md (hunk 3 above)
Comment (jpwalden) [reply Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md]: Valid point > removed

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+	* Use ```ctrl-C``` at the terminal to stop collecting the radio log and to close file logcat.txt.
+	* Visit oFono Logger app again and save the log as instructed in this article.
+
+* Export journal output into a file:
+	
+	```
+	journalctl -b -a --no-tail --no-pager > journal.txt
+	```
+	
+* Please find the log files logcat. txt and journal.txt in $HOME and ofono_*.tar.gz in $HOME/Documents. Let's package the 3 log files into a container:
```
Comment (jovirkku) [inline Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md:70]: I forgot to remove this sentence: " Please find the log files logcat. txt and journal.txt in $HOME and ofono_*.tar.gz in $HOME/Documents. ".  I think it is unnecessary now that we use the tar command (it does the finding for the user).

**Review by jovirkku — APPROVED** [review]

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md (hunk 4 above)
Comment (jpwalden) [reply Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md]: Thanks, true -that is not needed anymore > removed.

File: Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md (hunk 1 above)
Comment (vigejolla) [reply Support/Help_Articles/Collecting_Logs/Collect_SMS_Logs/README.md]: Well, `pkcon install sailfish-log-viewer-ofono` works on all devices. But the link is good.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**